tests/_v06/rfc5/transform_examples/test_transform_examples.py

```diff
@@ -0,0 +1,201 @@
+from pathlib import Path
+
+import pytest
+import zarr
+
+from ome_zarr_models._v06.image import Image
+
+from .expected_attributes import expected_attrs
```

Comment:
Code philosophy comment: I usually like to have full paths instead of relative ones for imports. Such as from `tests._v06.rfc5.transform_examples.expected_attributes`. 
VSCode (I'm using the GitHub webapp) wasn't able to resolve "Go To References" for `expected_attrs`. I wonder if it's because of the use of relative imports.
Any preference for the relative one, beside brevity?

Reply:
No reason, just being lazy - thanks for catching